canu/generate/switch/config/config.py

```diff
@@ -908,6 +936,60 @@ def vsx_mac(switch_name):
             variables["VSX_ISL_PORT1"] = pair_connections[0]
             variables["VSX_ISL_PORT2"] = pair_connections[1]
 
+    variables["NMN_RVR_VLANS"] = []
+    variables["HMN_RVR_VLANS"] = []
+    variables["NMN_RVR_VLAN_LIST"] = []
+    variables["HMN_RVR_VLAN_LIST"] = []
+
+    # Get RVR VLANs for spine and leaf switches
+    # This is ALL the RVR VLANS and IP addresses
+    if node_shasta_name in ["sw-spine", "sw-leaf"]:
+        nodes_by_name = {}
+        nodes_by_id = {}
+        for cabinets in (
+            sls_variables["NMN_RVR_CABINETS"] + sls_variables["HMN_RVR_CABINETS"]
+        ):
+            ip_address = netaddr.IPNetwork(cabinets["CIDR"])
```

Comment:
As in previous comment, I don't think we want to trust sls variables.  either catch an exception  here and exit with a good error message or make sure the sls file load goes through the sls_utils parser which does all this (look at report cabling code).